content/en/docs/Contribute/doc-con-formatting.md

```diff
@@ -30,9 +30,57 @@ Tekton markdown formatting conventions:
 
 -   **Markdown files**
     - Hard-wrap the file lines so they are not longer than 80 characters.
-    - Use [reference-style links][ref-links] if the URL in the link is too
+    - Use [reference-style links][ref-links] if the URL if the link is too

```

Comment:
'in' was correct here, although I had to read it a couple of times. We could probably just have:
`Use [reference-style links][ref-links] if the URL is too long.`

i.e. drop 'in the link'

Reply:
Oh, I think my brain ran into a race condition there 😅. Definitely no need to have both link and URL in that sentence.

Updated.